src/CookieStore.ts

```diff
@@ -1,5 +1,15 @@
 import { Cookie, parse as parseCookie } from 'set-cookie-parser'
 
+interface RequestLike {
+    credentials: 'include' | 'omit' | 'same-origin';
```

Comment:
Can we reference the `Request['credentials']` type here? For the sake of not repeating the union type.

Reply:
Good point. I've changed it.